pkg/install/images/integration/images_integration_test.go

```diff
@@ -73,7 +73,12 @@ func TestProcessImagesFromHelmChartsAndSystemApps(t *testing.T) {
 	if err != nil {
 		t.Errorf("Error calling GetImagesFromSystemApplicationDefinitions: %v", err)
 	}
+	defaultAppImages, err := images.GetImagesFromDefaultApplicationDefinitions(log, config, helmClient, 5*time.Minute, "")
```

Comment:
For some reason this fails to get the default app def and it reads the wrong path